bin/main.rb

```diff
@@ -1,2 +1,18 @@
 #!/usr/bin/env ruby
+
+require_relative '../lib/tictactoe'
+
 puts '"Hello world!"'
+
+new_game = Tictactoe.new
+new_game.execute
+
+puts 'want to play again?'
+answer = gets.strip.downcase
+
+if answer == 'yes'
+  game = Tictactoe.new
+  game.execute
+else
+  puts 'Hope to see you soon'
+end
```

Comment:
- [ ] All game-logic related resources are kept in the `lib` directory. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))